### PR TITLE
Fix invalid URL for pull request comments update/delete

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

**Comment by KostyaSha** [comment]
Some test case will be appreciated

**Comment by henryju** [comment]
I have added some integration tests and fixed regression from last commit in master.

**Comment by henryju** [comment]
Any update about this? Thanks

**Comment by oleg-nenashev** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <skipTests>true</skipTests>
```
Comment (KostyaSha) [inline pom.xml]: ???

Comment (henryju) [inline pom.xml]: Sorry, I have updated my own fork without thinking about this open P/R. I'll fix that.

**Comment by henryju** [comment]
Rebased on master for easier merge.

**Comment by kohsuke** [comment]
My apologies for the delay.

**Pull request merged**